tools/cockpit.spec

```diff
@@ -52,6 +52,10 @@ BuildRequires: docbook-style-xsl
 BuildRequires: keyutils-libs-devel
 BuildRequires: dbus-devel
 BuildRequires: glib-networking
+
+# There is a segfault in g_source_add_child_source that was fixed in
+# 2.37.3-2-g1d5c815
+BuildRequires: glib2-devel >= 2.40
```

Comment:
Yes, I was commenting on the BuildRequires line. Is that change necessary? The Requires: line is good.